src/frontend/citizen-portal/src/app/components/dispute-ticket-stepper/dispute-ticket-stepper.component.ts

```diff
@@ -119,13 +120,12 @@ export class DisputeTicketStepperComponent implements OnInit, AfterViewInit {
     this.form = this.formBuilder.group({
       ...this.ticketFormFields,
     });
-    this.form.get('country').setValue('Canada');
```

Comment:
This line must remain in or the field does not get initialized to Canada as per user requirements.

Reply:
otherwise PR looks good

Reply:
It is default in notice-of-dispute.service.ts
Or do you mean that is not working for default value?

Reply:
It is not working for default value without this line

Reply:
the code originally set for field disable. so it should be change from 
{ value: "Canada", disable: true } to { value: "Canada", disable: false } or "Canada"